cmd/testrunner.go

```diff
@@ -189,10 +189,14 @@ func testTypeCommandActionFactory(runner testrunner.TestRunner) cobraext.Command
 			return fmt.Errorf("cannot determine if package has data streams: %w", err)
 		}
 
-		runIndependentElasticAgent := false
+		// Enable independent agents if package defines that requires root privileges
```

Comment:
Maybe we can mention here that this is temporary till independent agents are enabled by default.